jsonschema-core.xml

```diff
@@ -1483,27 +1483,22 @@
                         <t>
                             Together with "$dynamicAnchor", "$dynamicRef" implements a cooperative
                             extension mechanism that is primarily useful with recursive schemas
-                            (schemas that reference themselves).  Both the extension point and the
-                            runtime-determined extension target are defined with "$dynamicAnchor",
-                            and only exhibit runtime dynamic behavior when referenced with
-                            "$dynamicRef".
+                            (schemas that reference themselves).  The extension point is defined with
```

Comment:
From my perspective the "extension point" phrase is misleading. In my understading the extension point is the $dynamicRef, since that is the point where a customizable/abstract schema makes itself open for extension. The $dynamicAnchor is the place where this extension point gets utilized, and the abstraction (the $dynamicRef attribute's value, which doesn't have semantics on its own) gets bound to a concrete value (the schema supplied by $dynamicAnchor).

I hope this makes sense. Please consider re-wording this section accordingly.

Reply:
I think the "extension point" language came from `$recursiveRef` when the only use case was extending recursive schemas. I think you're right that "extension point" doesn't make sense, at least not anymore. But, this has nothing to do with the change in this PR. I suggest a separate issue/PR to discuss improvements to how we express this feature. We can even get those improvements into the patch release ahead of this one.

Reply:
I think it does still make sense if you consider an "extension point" is the point at which you're extending the schema.

The anchor is the location you wish to make extensible. It feels odd because you reference it, but you're really extending that anchor location's schema.